packages/bits/src/lib/popup/popup.component.ts

```diff
@@ -70,7 +70,7 @@ import { PopupToggleDirective } from "./popup-toggle.directive";
     styleUrls: ["./popup.component.less"],
     encapsulation: ViewEncapsulation.None,
 })
-/* tslint:disable:use-host-property-decorator */
+/* eslint-disable */
```

Comment:
is there a rule name that could be specified here?

Reply:
added